test/__init__.py

```diff
@@ -0,0 +1,4 @@
+import sys, types
+sys.modules['pandas'] = types.ModuleType('pandas', 'Fake pandas module')
```

Comment:
Well, I do not wish to attempt to get pandas to build on the travis-ci host. 1) Testing a collaborative open source package against multiple Python versions requires something like travis-ci, and 2) using travis-ci means we have to build each test environment from scratch for each build. There are ways to retain assets from build to build on a TravisCI instance, but it requires a paid subscription.